scooby/report.py

```diff
@@ -1,21 +1,17 @@
 """The main module containing the `Report` class."""
 
 import importlib
+import importlib.metadata
 import sys
 import time
 from types import ModuleType
 
-from .knowledge import (
-    VERSION_ATTRIBUTES,
-    VERSION_METHODS,
-    get_filesystem_type,
-    in_ipykernel,
-    in_ipython,
-)
+from .knowledge import get_filesystem_type, in_ipykernel, in_ipython
 
 MODULE_NOT_FOUND = 'Module not found'
 MODULE_TROUBLE = 'Trouble importing'
 VERSION_NOT_FOUND = 'Version unknown'
+NOT_PROPERLY_INSTALLED = '(not properly installed)'
```

Comment:
This statement has a tone of judgement, and I would prefer scooby to not be opinionated about how users do things, I do not think it is the scope of scooby.

Having a local module is a valid scenario, and if a users decides to do so it is not helpful by always slapping in its face "not properly installed". E.g., a folder full of notebooks, with a small local "module"-folder in it that is accessed by all notebooks. This is only meant to be local, never used anywhere else. This might even be versioned. I think that happens more than you might think.

As such, I would prefer a short string, free of judgement. E.g., `(local)`, `(sys.path)`, or similar.